package.json

```diff
@@ -32,6 +32,7 @@
     "eslint-config-prettier": "^8.3.0",
     "ethereum-block-by-date": "^1.4.2",
     "ethers": "^5.1.0",
+    "lodash": "^4.17.21",
```

Comment:
I think we can use this elsewhere and it tree-shakes well. I don't have time tonight to do a bundle analysis, should be okay i think.